.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -7,147 +7,147 @@ name: Bug report
 description: Create a report to help us improve
 
 body:
-- type: markdown
-  attributes:
-    value: |
-      ⚠
-      Verify first that your issue is not [already reported on GitHub][issue search].
```

Comment:
Are the changes in this file simply changing indentation? If that's the case, can you please undo this and similar changes?

Removing whitespace at the end of lines is OK, but reindenting everything is definitely not, especially due to some fake "warning" that should better be ignored because we do not lint this stuff yet.

Reply:
Why is the reindenting not OK? Not only it should not affect any of the features of the repo, but it does bring the benefit of supressing the generation of unused content in every CI run. Not sure I see how that can be a bad thing.

Reply:
Because you write that it is related to the output of Galaxy importer. **We should ingore that output.** If this is a general unrelated formatting, it's OK (except potentially below), but if it is related to the Galaxy importer output, it is not OK.

Also we still haven't defined a YAML lint configuration. I don't want to have too many reformatting commits, and I don't want to reformat everything **again** once we do that. So if we reformat now, this should hopefully be the final reformat.

Reply:
That was spotted in the importer output, but the warnings are valid (well, a good part of them are, at least). I am not suggesting we monitor that output or anything, I am only saying that is easy for us to generate less content on every single run. Over time that means better use of resources, but I don't want to spend a lot of time on this either - I just saw a low hanging fruit and took it. If putting this through is going to be complicated, I am happy to drop it.

Reply:
No, let's merge this.